ads/_config.js

```diff
@@ -358,8 +358,6 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
-  'chargeads': {},
```

Comment:
You'll still need to keep the config here to be backward compatible. Could you please leave a comment wherever there's `chargeads` with an estimated date of deprecation. Thank you.

Reply:
Updated, will submit another PR in a month to remove chargeads